sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -25,70 +27,138 @@
 adjust_attribute_map()
 
 
-class QueueDescription(object):  # pylint:disable=too-many-instance-attributes
-    """Description of a Service Bus queue resource.
+def extract_kwarg_template(self, kwargs, extraction_missing_args, name):
+    if getattr(self, '_require_all_args', False):
```

Comment:
We don't need this any more, this was from when this was being used between multiple classes, some of which weren't fail-on-missing.

Reply:
Agree. I was also thinking of removing it.